tests/dummy/app/routes/index.js

```diff
@@ -40,7 +40,7 @@ export default Route.extend(RouteQueryManager, {
       {
         query: gql`
           query {
-            allDocuments {
+            allDocuments(form: "main") {
```

Comment:
Is "main" the default slug for `server.create("form")`?

Reply:
oops, that was not supposed to be in there - I just changed this locally for testing.

Reply:
Fixed.